packages/trader/src/AppV2/Components/ServicesErrorSnackbar/services-error-snackbar.tsx

```diff
@@ -29,11 +30,18 @@ const ServicesErrorSnackbar = observer(() => {
             return has_services_error;
         return false;
     };
+
     const should_show_error_snackbar = checkShouldShowErrorSnackBar();
+    const should_contain_action = should_show_error_snackbar && code === SERVICE_ERROR.COMPANY_WIDE_LIMIT_EXCEEDED;
     const bottom_position =
         location.pathname.startsWith('/contract/') && is_multiplier && isValidToCancel(contract_info)
             ? '104px'
             : '48px';
+    const action_props = {
+        actionText: localize('View'),
+        delay: 8000,
```

Comment:
What is this magic number?

Reply:
The default delay is 4000 (=4s). As per design this snackbar (with long message and link) should stay for a longer period of time - 8s